src/System.Runtime/src/System/IO/FileStream.cs

```diff
@@ -0,0 +1,49 @@
+namespace System.IO {
+    public class FileStream : System.IO.Stream
+    {
+        [Obsolete("This constructor has been deprecated.  Please use new FileStream(SafeFileHandle handle, FileAccess access) instead.  http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
What does stubbing this out help with? As it is still going to throw at runtime.